krita_plugin/krita_diff/krita_diff.py

```diff
@@ -137,8 +139,8 @@ def img2img(self, path, mask_path, mode):
             "mode": mode,
             "src_path": path,
             "mask_path": mask_path,
-            "prompt": self.fix_prompt(
-                self.cfg('img2img_prompt', str) if not self.cfg('img2img_prompt', str).isspace() else None),
+            "prompt": self.cfg('img2img_prompt', str) if not self.cfg('img2img_prompt', str).isspace() else None,
```

Comment:
I think it's worth to leave fix_prompt function here. This function replaces new lines with commas and removes redundant empty lines from prompt. I don't think there is a real difference for a model, but it makes logs more readable.